### DISCUSSION TITLE
How can we set  the number of CPU threads in CAFE5?

### DISCUSSION THREAD

**Discussion by ymat2** [hahnlab/CAFE5#113] [Q&A] [answered]
In the previous version, we can control # of threads by `-t` option like:
```
load -i filename -t 5 -l logfile -p 0.05
```

Can we set # of CPU threads when running CAFE5 analysis?

**Comment by benfulton** [comment, marked as answer]
Yes, use the --cores argument.

`cagee -t example_tree.txt -i test.tsv --cores 5
`
